GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -31,13 +31,12 @@ public virtual bool TryRunPreUpgradeChecks(out string consoleError)
                 if (this.IsUnattended())
                 {
                     consoleError = $"{GVFSConstants.UpgradeVerbMessages.GVFSUpgrade} is not supported in unattended mode";
-                    this.tracer.RelatedError($"{nameof(this.TryRunPreUpgradeChecks)}: {consoleError}");
+                    this.tracer.RelatedWarning($"{nameof(this.TryRunPreUpgradeChecks)}: {consoleError}");
                     return false;
                 }
 
                 if (!this.IsGVFSUpgradeAllowed(out consoleError))
                 {
-                    this.tracer.RelatedError($"{nameof(this.TryRunPreUpgradeChecks)}: {consoleError}");

```

Comment:
Does this change the user experience?  I.e. previously they would have gotten the message on their screen.

Reply:
No it does not change user experience.